CMakeLists.txt

```diff
@@ -2301,27 +2301,32 @@ endif()
 # FFmpeg 4.x support
 # FFmpeg is multimedia library that can be found http://ffmpeg.org/
 find_package(FFMPEG COMPONENTS libavcodec libavformat libavutil libswresample)
-cmake_dependent_option(FFMPEG "FFmpeg 4.x support" ON "FFMPEG_FOUND;FFMPEG_libavcodec_VERSION VERSION_GREATER_EQUAL 58" OFF)
+# On Windows version numbers are not available!
```

Comment:
Is this a bug in our windows build environment?

Reply:
perhaps a bug in vcpkg